nuget.config

```diff
@@ -1,9 +1,10 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="AzureLatest" value="https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json" protocolVersion="3" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="MSBuild-SDKs" value="https://myget.org/F/msbuild-sdks/api/v3/index.json" />
```

Comment:
What's the feed for?

Reply:
See the comment above. It is temporary, just to get the solution building properly!